src/utils/mtev_lfu.c

```diff
@@ -78,19 +83,25 @@ hs_init(ck_hs_t *hs, unsigned int mode, ck_hs_hash_cb_t *hf, ck_hs_compare_cb_t
 }
 
 static bool
-hs_string_compare(const void *a, const void *b)
+hs_lfu_key_compare(const void *a, const void *b)
```

Comment:
not sure if this is right.... it would return `b` < `aa`, and I'm not sure that's what we want.